src/domain/home/form/HomePreliminaryForm.tsx

```diff
@@ -161,9 +143,13 @@ const actions = {
   setFormValues,
 };
 
+const mapStateToProps = (state: StoreState) => ({
+  stateFormValues: state.registration.formValues,
```

Comment:
should be named `initialFormValues`

Reply:
But they are not initial, they are from state..

Reply:
..and I thought it would be confusing to use the same variable name as a "default" object.

Reply:
uhm that make sense